qa-pipelines/config-azure.yml

```diff
@@ -26,5 +26,5 @@ organization: splatform
 
 magic-dns-service: omg.howdoi.website
 
-cap-sle-url: https://s3.amazonaws.com/cap-release-archives/master/scf-sle-2.11.0%2Bcf1.36.0.0.g4c9e9ae1.zip
-cap-opensuse-url: https://s3.amazonaws.com/cap-release-archives/master/scf-opensuse-2.11.0%2Bcf1.36.0.0.g4c9e9ae1.zip
+cap-sle-url: https://s3.amazonaws.com/cap-release-archives/master/scf-sle-2.13.3%2Bcf2.7.0.0.gf95d9aed.zip
+cap-opensuse-url: https://s3.amazonaws.com/cap-release-archives/master/scf-opensuse-2.13.3%2Bcf2.7.0.0.gf95d9aed.zip
```

Comment:
Having the URLs in a config (rather than a resource that just grabs the latest release / master, or something) seems odd, but :man_shrugging:

Reply:
Its the specific release from which we need to upgrade from. we don't grab it from resource because all RCs goes in /master/ . We also use this as, kind of a hack for any custom deployments where we need a specific build without doing much change to the code base